src/Config.php

```diff
@@ -364,7 +364,12 @@ protected function parseContentTypes(array $generalConfig)
      */
     protected function parseTheme($themePath, array $generalConfig)
     {
-        $themeConfig = $this->parseConfigYaml('config.yml', $themePath);
+        $themeConfig = $this->parseConfigYaml('theme.yml', $themePath);
+
+        /** @deprecated Since 2.3 and will be removed in Bolt v3.0 (config.yml was the old filename) */
```

Comment:
Ah, no.. I used the checkout from _before_ the version bump to do the backport.. Will fixe these numbers.